Exercise-01/Exercise01/FirstProblem.swift

```diff
@@ -25,7 +25,44 @@ import Foundation
 ///   - player: the player representation used to fill the square.
 ///   - board: the board to be modified.
 func readNextMove(player: String, board: inout Board) {
-  // TODO: implement this function
+    var done = false
+    while !done {
+        print("\(player) turn, what is your move?")
+        
+        let line = readLine()
+        guard line != nil else {
```

Comment:
```suggestion
        guard let line = readLine() else 
```
this is much more readable and ...